man/systemd-id128.xml

```diff
@@ -183,16 +183,17 @@ $ systemd-id128 machine-id -u
 $ systemd-id128 new -u
 1fb8f24b-02df-458d-9659-cc8ace68e28a
 
-$ systemd-id128 --app=1fb8f24b-02df-458d-9659-cc8ace68e28a
+$ systemd-id128 machine-id -u --app=1fb8f24b-02df-458d-9659-cc8ace68e28a
```

Comment:
Then, for consistency, `-u` -> `--uuid` is better here, above, and below ?

Reply:
I didn't want to use the long forms everywhere, because then this all becomes very verbose. I changed `--app` to `-a` now. (`--app` works, because popt allows abbreviations, but I guess it's better to not document this.)